### DISCUSSION TITLE
I see the word 'devland' being used in Quasar docs. What does it mean?

### DISCUSSION THREAD

**Discussion by Ammamon** [quasarframework/quasar#8954] [General - Components / Directives / etc] [answered]
I see the word 'devland' being used in Quasar docs/forums. What does it mean?

For eg:

"There have been a number of recent exploits (especially for older Android and iOS devices) where certain emoji and non-standard UTF-8 actually triggered mobile device restarts and boot-screen loops. DO consider a **devland** integration of markdown parsing in a plain-text type of input field and render it to HTML on the server side before you pass it to the chat recipients."

"A draggable qdialog  can be done with TouchPan directive. The problem with doing this within the framework is that we don't know what the Dialog will contain so we can't just attach the logic to the whole Dialog. Some content's behavior might negatively interfere with TouchPan. However, in **devland**, you can use the directive on a particular part of the dialog (like a QToolbar for the title or a specific QCardSection)."

**Comment by hawkeye64** [comment, marked as answer]
@Ammamon 
userland - the people that use your product
devland - the developers that make a product
creatorland - the developers that make a component/library/framework that you consume in devland

**Reply by Ammamon** [reply]
Thank you very much for the explanation.
